lowrisc-toolchain-gcc-rv32imc.config

```diff
@@ -1,9 +1,13 @@
 CT_CONFIG_VERSION="3"
 CT_EXPERIMENTAL=y
+
 CT_ARCH_RISCV=y
-CT_ARCH_ARCH="rv32imcb"
+# This toolchain will support -march=rv32imcb but the libraries are not built
+# with the B extension enabled.
+CT_ARCH_ARCH="rv32imc"
```

Comment:
To clarify here: CT_ARCH_ARCH is the *minimum* set of ISA extension the processor needs to support? This could be added to the comment.

Reply:
Can clarify once a preliminary build is complete.